DP03_05_Near-Earth_Objects.ipynb

```diff
@@ -0,0 +1,1395 @@
+    "ax.annotate('retrograde', xy=(0.05, 95.0), xytext=(0.05, 95.0))\n",
+    "plt.xlabel(\"semimajor axis (au)\")\n",
+    "plt.ylabel(\"inclination (deg)\")\n",
+    "plt.title(\"NEOs in DP0.3\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d150c3ac-e906-440d-b18d-5105c642a702",
+   "metadata": {},
+   "source": [
+    "The retrograde (i>90 deg) NEOs in the DP0.3 catalogs can be extracted and plotted with their object designations labeled."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "d4be9f24-da4d-4e97-b302-85e2b97e7b3d",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "high_i = (uniqueNEOs.incl > 90.0) & (uniqueNEOs.q < 1.3)\n",
+    "print(len(uniqueNEOs[high_i]))\n",
+    "print(uniqueNEOs[high_i][['mpcDesignation', 'q', 'a', 'e', 'incl']])"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "7c4561e5-2447-43ca-baf9-5582cf0cff86",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "fig, ax = plt.subplots()\n",
+    "plt.xlim([0.,4.])\n",
+    "plt.ylim([90.,180.])\n",
+    "ax.scatter(uniqueNEOs[high_i].a, uniqueNEOs[high_i].incl)\n",
+    "ax.annotate(uniqueNEOs[high_i].iloc[0].mpcDesignation, xy=(uniqueNEOs[high_i].iloc[0].a, uniqueNEOs[high_i].iloc[0].incl), xytext=(uniqueNEOs[high_i].iloc[0].a+0.1, uniqueNEOs[high_i].iloc[0].incl-2.), fontsize=10)\n",
+    "ax.annotate(uniqueNEOs[high_i].iloc[1].mpcDesignation, xy=(uniqueNEOs[high_i].iloc[1].a, uniqueNEOs[high_i].iloc[1].incl), xytext=(uniqueNEOs[high_i].iloc[1].a-0.65, uniqueNEOs[high_i].iloc[1].incl-2.), fontsize=10)\n",
+    "ax.annotate(uniqueNEOs[high_i].iloc[2].mpcDesignation, xy=(uniqueNEOs[high_i].iloc[2].a, uniqueNEOs[high_i].iloc[2].incl), xytext=(uniqueNEOs[high_i].iloc[2].a+0.1, uniqueNEOs[high_i].iloc[2].incl-2.), fontsize=10)\n",
+    "ax.annotate(uniqueNEOs[high_i].iloc[3].mpcDesignation, xy=(uniqueNEOs[high_i].iloc[3].a, uniqueNEOs[high_i].iloc[3].incl), xytext=(uniqueNEOs[high_i].iloc[3].a+0.1, uniqueNEOs[high_i].iloc[3].incl-2.), fontsize=10)\n",
+    "ax.annotate(uniqueNEOs[high_i].iloc[4].mpcDesignation, xy=(uniqueNEOs[high_i].iloc[4].a, uniqueNEOs[high_i].iloc[4].incl), xytext=(uniqueNEOs[high_i].iloc[4].a+0.1, uniqueNEOs[high_i].iloc[4].incl-2.), fontsize=10)\n",
+    "ax.annotate(uniqueNEOs[high_i].iloc[5].mpcDesignation, xy=(uniqueNEOs[high_i].iloc[5].a, uniqueNEOs[high_i].iloc[5].incl), xytext=(uniqueNEOs[high_i].iloc[5].a+0.1, uniqueNEOs[high_i].iloc[5].incl-2.), fontsize=10)\n",
+    "plt.xlabel(\"semimajor axis (au)\")\n",
+    "plt.ylabel(\"inclination (deg)\")\n",
+    "plt.title(\"Retrograde NEOs in DP0.3\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "5a54f3e2-8c81-482d-a023-faa4eb895fda",
+   "metadata": {},
+   "source": [
+    "### Exercises to try:\n",
+    "\n",
+    "#### Plot the subset of Apollos that are only Venus-crossing (i.e., exclude the Earth-crossing Apollos).\n",
```

Comment:
Do we want these to be in normal format (not bold?)

Reply:
No longer in bold.